abr_control/controllers/signals/avoid_joint_limits.py

```diff
@@ -13,22 +13,51 @@ class AvoidJointLimits(Signal):
     Parameters
     ----------
     min_joint_angles : np.array
-      the lower bound on joint angles [radians]
+        the lower bound on joint angles [radians]
     max_joint_angles : np.array
-      the upper bound on joint angles [radians]
+        the upper bound on joint angles [radians]
     max_torque : np.array, optional (Default: 1)
-      the maximum force to push away with [Nm]
+        the maximum force to push away with [Nm]
+    cross_zero: List of Boolean, Optional
```

Comment:
can we detect this based on the min max?

Reply:
I don't think you can in every case. I remember this being a major pain to get working. I'll have to check through my notes to make sure.

Reply:
alright yeah, it would be good to have more information on why this is necessary if it is (in the comment)

Reply:
It's so you can use the same conditional statements to determine when to use the opposing force and in what direction to push. I'll see if I can summarize it easily